examples/moonsniff/tbbmatch.lua

```diff
@@ -0,0 +1,379 @@
+--- Matching for pcap files
+
+local mod = {}
+
+local memory    = require "memory"
+local ts        = require "timestamping"
+local hist      = require "histogram"
+local log       = require "log"
+local ms	= require "moonsniff-io"
+local dpdk	= require "dpdk"
+local pcap	= require "pcap"
+local hmap	= require "hmap"
+
+local ffi    = require "ffi"
+local C = ffi.C
+
+-- pointers and ctypes
+local CHAR_P = ffi.typeof("char *")
+local INT64_T = ffi.typeof("int64_t")
+local UINT64_P = ffi.typeof("uint64_t *")
+local UINT8_T = ffi.typeof("uint8_t")
+local UINT8_P = ffi.typeof("uint8_t*")
+
+local free = C.rte_pktmbuf_free_export
+local band = bit.band
+
+local pktmatch = nil
+local scratchpad = nil
+local SCR_SIZE = 16 		-- size of the scratchpad in bytes, must always be multiple of 8 for hash to work
+				-- maximum: 64 (largest supported key size for hashmap)
+local mempool = nil
+local mempool2 = nil
+local next_mem = 0
```

Comment:
nextMem?

Reply:
Used to switch between mempools

Renamed mempools and next_mem